don't put the online help in /usr/share/man/help

--- a/ved/ved.h
+++ b/ved/ved.h
@@ -165,7 +165,7 @@
 #define	HELPFILE	UC "/doc/cmds/ved.help"
 #else
 #define	DEFSHELL	"/bin/sh"
-#define	HELPFILE	UC "/opt/schily/share/man/help/ved.help"
+#define	HELPFILE	UC "/opt/schily/share/doc/ved/help/ved.help"
 #endif
 
 /*
--- a/ved/ved.c
+++ b/ved/ved.c
@@ -530,7 +530,7 @@
 	Uchar	*hfile = HELPFILE;
 	Uchar	*name;
 
-	name = _gethelpfile("share/man/help/ved.help");
+	name = _gethelpfile("share/doc/ved/help/ved.help");
 	if (name == NULL)
 		name = _gethelpfile("man/help/ved.help");
 
--- a/ved/ved.1
+++ b/ved/ved.1
@@ -3276,7 +3276,7 @@
 .B ved
 shell command execute buffer.
 .TP
-.B .../man/help/ved.help
+.B .../doc/ved/help/ved.help
 The on-line help file is assumed to be in the same
 directory tree as the
 .B ved
--- a/ved/Makefile.help
+++ b/ved/Makefile.help
@@ -7,7 +7,7 @@
 include		$(SRCROOT)/$(RULESDIR)/rules.top
 ###########################################################################
 
-INSDIR=		share/man/help
+INSDIR=		share/doc/ved/help
 TARGET=		ved.help
 #XMK_FILE=	Makefile.man
 
